static/app/components/organizations/headerItem.tsx

```diff
@@ -129,6 +129,7 @@ const StyledHeaderItem = styled('div', {
 const Content = styled('div')`
   display: flex;
   flex: 1;
+  width: 0;
```

Comment:
Since this has flex-grow: 1 this is fine and prevents the dropdown content from overflowing: 


<img width="436" alt="image" src="https://user-images.githubusercontent.com/10888943/192648179-b9120251-ef57-45cd-8311-693eaa6a3928.png">